src/pykiso/lib/connectors/cc_virtual_can.py

```diff
@@ -0,0 +1,138 @@
+##########################################################################
+# Copyright (c) 2010-2022 Robert Bosch GmbH
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0
+##########################################################################
+
+"""
+Virtual Can Communication Channel
+*********************************
+
+:module: cc_virtual_can
+
+:synopsis: CChannel implementation for victual CAN using UDP multicast API from python-can
+
+.. currentmodule:: cc_virtual_can
+
+"""
+import logging
+from typing import Dict, Optional, Union
+
+from pykiso import connector
+
+try:
+    import can
+    from can.interfaces.udp_multicast.bus import UdpMulticastBus
+except ImportError as e:
+    raise ImportError(
+        f"{e.name} dependency missing, consider installing pykiso with 'pip install pykiso[can]'"
+    )
+
+log = logging.getLogger(__name__)
+
+
+class CCVirtualCan(connector.CChannel):
+    def __init__(
+        self,
+        channel: UdpMulticastBus = UdpMulticastBus.DEFAULT_GROUP_IPv4,
+        interface: str = "udp_multicast",
```

Comment:
This shouldn't be part of the parameters, should it?
It feels like if this is changed, the entire connector is broken